src/main/java/com/dataloom/graph/edge/EdgeKey.java

```diff
@@ -31,22 +29,26 @@
  *
  */
 public class EdgeKey {
-    private final UUID srcVertexId;
+    private final UUID srcEntityKeyId;
+    private final UUID dstEntityKeyId;
     private final UUID dstTypeId;
     private final UUID edgeTypeId;
-    private final UUID dstVertexId;
     private final UUID edgeEntityKeyId;
 
-    public EdgeKey( UUID srcVertexId, UUID dstTypeId, UUID edgeTypeId, UUID dstVertexId, UUID edgeEntityKeyId ) {
-        this.srcVertexId = srcVertexId;
+    public EdgeKey( UUID srcEntityKeyId, UUID dstEntityKeyId, UUID dstTypeId, UUID edgeTypeId, UUID edgeEntityKeyId ) {
```

Comment:
It may be better to order the five fields in the same order of `Table.java`, so that we don't need to worry about wrong order of columns